src/pylife/materialdata/woehler/elementary.py

```diff
@@ -14,14 +14,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# ----------------------------------------------------
+# Matplus GmbH altered the code formatting and removed Python 
+# libraries such as Matplotlib and pandas to integrate pyLife into EDA. 
+# There are no changes in the functionality of the pyLife modules.
+# ----------------------------------------------------
+
 import numpy as np
 import pandas as pd
 import scipy.stats as stats
 
 from .likelihood import Likelihood
 from .pearl_chain import PearlChainProbability
-import pylife.utils.functions as functions
-from . import FatigueData, determine_fractures
+import local_pyLife.utils.functions as functions
```

Comment:
This does not work like that. The package is still called `pylife`. In a proper pylife installation this will outright fail, like in our test suite: See [here](https://github.com/boschresearch/pylife/actions/runs/3376774688/jobs/5604902007#step:5:29).